shared/clinical-code-sets/tests/ferritin/1/README.md

```diff
@@ -0,0 +1,15 @@
+# Ferritin
+
+A patient's ferritin as recorded via clinical code and value. This code set only includes codes that are accompanied by a value (`XaItW - Plasma ferritin level`). It does not include codes that indicate a patient's ferritin without giving the actual value.
+
+Ferritin codes retrieved from [GetSet](https://getset.ga) and metadata available in this directory.
```

Comment:
An example of the metadata from GetSet can be found in the file `copd/1/copd.ctv3.metadata.json`. This type of file isn't currently in this directory so I suggest remove this line of the README. It can perhaps be added back in once getset is back online and you can download the metadata files from there.